src/resources/dicts/fr.edn

```diff
@@ -435,25 +435,25 @@
     :command.sidebar/close-top "Ferme l'élément supérieur dans la barre latérale droite"
     :export-copied-to-clipboard "Copié dans le presse-papier"
     :code "Code"
-    :loading "Chargement..."
+    :loading "Chargement…"
```

Comment:
Looks like these ellipsis were all changed. If this is a personal preference it may be helpful to ping other French contributors to see if they also want the change

Reply:
It's not only a personal preference, it's the normal typography in French. For example, MS Words or LibreOffice change automatically '...' to '…'.
You're right, all French contributors should be aware about this.

Reply:
…and the normalised french disposition (bépo) allows typing them directly and «»ÉÈÀÇæœÆŒ also.